lib/core/test/unit/Cardano/Wallet/Primitive/ModelSpec.hs

```diff
@@ -110,15 +108,6 @@ spec = do
         it "applyBlock moves the current tip forward"
             (property prop_applyBlockCurrentTip)
 
-        it "Wallet starts with a block height of 0"
-            (property prop_initialBlockHeight)
-
-        it "applyBlock increases the blockHeight"
-            (property prop_applyBlockBlockHeight)
-
-        it "applyBlocks increases the blockHeight"
-            (property prop_applyBlocksBlockHeight)
```

Comment:
Can't we keep these last two actually? Applying blocks should increase the tip's height...

Reply:
That is arguably more a test of the test data than of any code, but I think it makes sense still.

There are gaps in the blockHeights of the test data, so we cannot assert that the height increases by `+1` (even if treating genesis as a special case).

So I relaxed the property to say that the blockHeight must increase (`bh' > bh`)